docker-compose.yml

```diff
@@ -377,29 +375,32 @@ services:
     # L3VXLAN Symmetric IRB:
     #   Ping to Bleaf internet connectivity IPs from Host1 via green and yellow VRFs from Leaf1
     command: |
-      sh -x -c 'sleep 30 && \
-                ip link add link eth0 name eth0.10 type vlan id 10 && \
+      sh -x -c 'ip link add link eth0 name eth0.10 type vlan id 10 && \
                 ip link set eth0 up && ip addr flush dev eth0 && \
                 ip link set eth0.10 up && \
                 ip addr add 10.10.10.10/24 dev eth0.10 && \
+                sleep 5 && \
                 ping -c 3 10.10.10.11 && \
-                ip link add link eth1 name eth1.20 type vlan id 20 && \
-                ip link set eth1 up && ip addr flush dev eth1 && \
-                ip link set eth1.20 up && \
-                ip addr add 20.20.20.20/24 dev eth1.20 && \
+                ip link add link eth0 name eth0.20 type vlan id 20 && \
+                ip link set eth0 up && ip addr flush dev eth0 && \
+                ip link set eth0.20 up && \
+                ip addr add 20.20.20.20/24 dev eth0.20 && \
                 ip route replace default via 20.20.20.1 && \
-                ping -c 3 30.30.30.31 && ping -c 3 5.5.5.5 && \
-                ip link add link eth2 name eth2.40 type vlan id 40 && \
-                ip link set eth2 up && ip addr flush dev eth2 && \
-                ip link set eth2.40 up && \
-                ip addr add 40.40.40.40/24 dev eth2.40 && \
+                sleep 5 && \
+                ping -c 3 30.30.30.31 && \
+                ip link add link eth0 name eth0.40 type vlan id 40 && \
+                ip link set eth0 up && ip addr flush dev eth0 && \
+                ip link set eth0.40 up && \
+                ip addr add 40.40.40.40/24 dev eth0.40 && \
                 ip route replace default via 40.40.40.1 && \
-                ping -c 3 6.6.6.6 && \
-                ip link add link eth3 name eth3.50 type vlan id 50 && \
-                ip link set eth3 up && ip addr flush dev eth3 && \
-                ip link set eth3.50 up && \
-                ip addr add 50.50.50.50/24 dev eth3.50 && \
+                sleep 5 && \
+                ping -c 3 6.6.6.6 && ping -c 3 40.40.40.41 && \
+                ip link add link eth1 name eth1.50 type vlan id 50 && \
+                ip link set eth1 up && ip addr flush dev eth1 && \
```

Comment:
eth1 should be access link, will fix the issue in the future PR. Refer the issue for tracking; https://github.com/opiproject/opi-evpn-bridge/issues/78